core_backend/app/question_answer/routers.py

```diff
@@ -292,7 +322,7 @@ async def content_feedback(
                 },
             )
         await update_votes_in_db(
-            user_id=USER1_USERNAME,  # TEMPORARY HARDCODED USER ID
+            user_id=user.user_id,
```

Comment:
Ooh. I guess you need it for content_feedback. Do you think it's worth adding to `response_feedback` for consistency?

Reply:
Adding the "user_id" column you mean?

Reply:
Keeping a note of this.